PocketKit/Sources/Analytics/Tracker/LinkedTracker.swift

```diff
@@ -20,8 +20,8 @@ class LinkedTracker: Tracker {
         parent.track(event: event, self.contexts + additional)
     }
 
-    func track(event: Event) {
-        parent.track(event: event)
+    func track(event: Event, filename: String = #file, line: Int = #line, column: Int = #column, funcName: String = #function) {
```

Comment:
Passing through the original functions/files for logging purposes. Most utility catch all classes will need this for logging.